tests/integrationtest/t/planner/core/integration.test

```diff
@@ -2000,18 +2000,6 @@ select * from t where a;
 set @@tidb_opt_range_max_size=DEFAULT;
 
 
-# TestIssue38295
```

Comment:
I moved back this test, because the result (error message content) is unstable. Some times it gives error for `c1`, but sometimes return error for `c2`.